scvi/hub/_utils.py

```diff
@@ -0,0 +1,24 @@
+import os
+from typing import Optional
+
+import anndata
+
+from scvi.data._utils import _is_latent
+
+
+def _get_counts_and_latent_info(
+    local_dir: str,
+    cell_count: Optional[int] = None,
+    gene_count: Optional[int] = None,
+    is_latent: Optional[bool] = None,
+):
+    if os.path.isfile(f"{local_dir}/adata.h5ad"):
+        adata = anndata.read_h5ad(f"{local_dir}/adata.h5ad", backed=True)
+        return adata.n_obs, adata.n_vars, _is_latent(adata)
```

Comment:
there's probably a more efficient way to do this, using the h5 api, or read elem from anndata

Reply:
Added to https://github.com/scverse/scvi-tools/issues/1817
See https://github.com/scverse/scvi-tools/pull/1812#discussion_r1045147118